src/lucky_task/task.cr

```diff
@@ -279,5 +279,5 @@ abstract class LuckyTask::Task
   end
 
   abstract def call
```

Comment:
I can't add a return type to this method since it can return anything you want. The specs currently rely on the return value of this to help test.